samples/tab-personal-quickstart/js/package.json

```diff
@@ -6,15 +6,15 @@
   },
   "private": true,
   "dependencies": {
-    "@microsoft/teams-js": "^1.6.0",
     "@fluentui/react-northstar": "^0.49.0",
+    "@microsoft/teams-js": "^2.0.0-beta.6",
```

Comment:
2.0 GA should be on npm by eow. We can update this to GA once its available on npm.

Reply:
Yes changed to latest GA 2.0 version